packages/mobile_app/lib/main.dart

```diff
@@ -16,6 +17,7 @@ void main() {
 
 class ProphetApp extends StatelessWidget {
   final usersRepository = UsersRepositoryFlutter();
+  final userPoll = UserPoll(mood: 3);
```

Comment:
Number 3 is ... ? (Worth adding a comment at least)

Reply:
Default value, I'll add comment

Reply:
Done